helm/azure-operator-chart/templates/rbac.yaml

```diff
@@ -9,6 +9,12 @@ rules:
       - customresourcedefinitions
     verbs:
       - "*"
+  - apiGroups:
+      - core.giantswarm.io
+    resources:
+      - drainerconfigs
+    verbs:
+      - "*"
```

Comment:
We drain guest cluster nodes on updates now. So we have to allow the management of drainer configs.